notebooks/3_KernelMethods.ipynb

```diff
@@ -35,9 +35,13 @@
     "from utilities.plotting import (\n",
     "    plot_projection, plot_regression, check_mirrors, get_cmaps, table_from_dict\n",
     ")\n",
-    "from utilities.kernels import linear_kernel, gaussian_kernel, center_kernel\n",
-    "from utilities.classes import PCA, LR, KPCA, KRR, KPCovR\n",
-    "\n",
+    "from utilities.kernels import linear_kernel, gaussian_kernel,center_kernel # to be replaced with scalers from skcosmo\n",
```

Comment:
scalers?

Reply:
It should be "to be replaced with the functions from skcosmo".